### PR TITLE
Add Centos 8 and Ubuntu 20.04

### PR DESCRIPTION
**Pull request by manics**
I've had the Centos 8 branch around for a while. Since there's a [request for Ubuntu 20.04](https://forum.image.sc/t/install-omero-w-ansible-on-ubuntu-20-04/38543) I thought I'd add it and open this PR.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,15 +6,24 @@ services:
   - docker
 
 install:
-  - pip install "ome-ansible-molecule<0.5"
-
+  - >-
+      if [ "$PYTHON" = "2" ]; then
+          pip install "ome-ansible-molecule<0.5";
+      fi
+  # Ansible 2.6 has problems when the remote python version is 3
```
Comment (sbesson) [inline .travis.yml]: Now that the work for unifying up and testing all the [OME Ansible roles](https://github.com/ome/ansible-roles) using `ome-ansible-molecule` has been completed, is it worth coming back to https://github.com/ome/ome-ansible-molecule/pull/20 and move towards a new release of the meta-package?

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [ANSIBLE-ROLES-merge#1173](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1173/). See the [console output](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/1173/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - meta/main.yml

--conflicts

**Comment by sbesson** [comment]
Given the interest in Ubuntu 20.04 in particular, this might be something worth re-activating with GitHub actions.

**Review by sbesson — APPROVED** [review]
Belated thanks. Merging and tagging as 4.3.0 as we are trying to complete the ongoing contribution for supporting Ubuntu 20.04

**Pull request merged**